ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -94,7 +94,8 @@ class AnalysisSectionIndirect(AnalysisSectionBase):
     partofday: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
     performance: str = "diff_time"  # "diff_time" or "diff_dist" relates to the used criticality metric
     resilience_curve_file: str = ""
-    disruption_steps_file: str = ""
+    traffic_intensities_file: str = ""
+    values_of_time_file: str = ""
```

Comment:
Paths should have the `Path` type hint
```suggestion
    traffic_intensities_file: Optional[Path] = None
    values_of_time_file: Optional[Path] = None
```